indico/modules/categories/templates/management/_events_list.html

```diff
@@ -0,0 +1,128 @@
+{% from 'pagination.html' import render_pagination %}
+
+{% macro event_row(event) %}
+    <tr class="i-table">
+        <td class="i-table column-icon">
+            <input type="checkbox" name="event_id" value="{{ event.id }}">
+        </td>
+        <td class="i-table column-icon">
+            {% if event.is_self_protected %}
+                <span class="icon-lock"></span>
+            {% elif event.is_public %}
+                <span class="icon-unlocked"></span>
+            {% endif %}
+        </td>
+        <td class="i-table">
+            <a href="{{ url_for('event_mgmt.conferenceModification-managementAccess', event) }}">
+                {{ event.title }}
+            </a>
+        </td>
+        <td class="i-table">{{ event.start_dt|format_datetime(timezone=event.timezone) }}</td>
+        <td class="i-table actions">
+            <a href="#"
+               class="i-button-icon icon-remove"
+               title="{% trans %}Delete event{% endtrans %}"
+               data-href="{{ url_for('event_management.delete', event) }}"
+               data-ajax-dialog
+               data-reload-after></a>
+            <a class="i-button-icon icon-transmission"
+               title="{% trans %}Move to different category{% endtrans %}"></a>
+            <a href="{{ url_for('event.conferenceDisplay', event) }}"
+               class="i-button-icon icon-arrow-right"
+               title="{% trans %}Go to the event display page{% endtrans %}"></a>
+        </td>
+    </tr>
+{% endmacro %}
+
+{% macro events_table(category, page, order_column, direction, events) %}
+    <table class="i-table event-management">
+        <thead>
+            <tr class="i-table">
+                <th class="i-table column-icon"></th>
+                <th class="i-table column-icon"></th>
+                <th class="i-table sortable-column {{ direction if order_column == 'title' else ''}}">
+                    <a href="{{ url_for('.manage_content', category, page=page,
```

Comment:
I have the feeling someone will end up writing a helper class for pagination+sorting at some point. ;) Doesn't necessarily need to be now - the simple approach works fine. But as a bonus once everything else is done or after the release..